packages/chat-component-bindings/src/utils/updateMessagesWithAttached.ts

```diff
@@ -26,8 +26,16 @@ export const updateMessagesWithAttached = (chatMessagesWithStatus: Message[]): v
     const previousSenderId = previousMessage?.messageType === 'chat' ? previousMessage.senderId : undefined;
     const nextSenderId = nextMessage?.messageType === 'chat' ? nextMessage.senderId : undefined;
 
+    const timediff =
+      new Date(message?.createdOn ?? '').getTime() - new Date(previousMessage?.createdOn ?? '').getTime();
+
+    const diffMins = Math.round(timediff / 1000 / 60); // minutes
```

Comment:
Add a value to constants.ts 
like 
const MINUTE = 1000*60;

Reply:
or MINUTE_IN_MS

Reply:
updated